### PR TITLE
Add link click and page load pixels

### PR DESCRIPTION
**Pull request by meltuhamy**
This PR adds engagement pixels for link click and page load events

### CODE REVIEW

File: scripts/pixels.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  // --------------------
+
+  // Link Clicks
+  // -----------
+  // Fire pixels when links within the article are clicked.
+  // Example: helppages.link.quora.https-spreadprivacy-com.google-filter-bubble-study
+  Array.prototype.slice.call(document.querySelectorAll('article a'), 0).forEach(function (link) {
+    link.addEventListener('click', function () {
+      firePixel('link', sanitizeUrl(source), sanitizeUrl(pathname), sanitizeUrl(link.href), {
+        once: false
```
Comment (moollaza) [inline scripts/pixels.js:97]: This should be true. By default, we don't want duplicate pixels

```suggestion
        once: true
```

Comment (meltuhamy) [reply scripts/pixels.js:97]: Do we not want to fire the pixels if someone clicks on the link twice or more [like we do on the blog](https://github.com/duckduckgo/Casper/blob/master/assets/js/pixels.js#L207)?

Comment (moollaza) [reply scripts/pixels.js:103]: I would say that’s a mistake to have done that for the blog. By default we don’t do this, as it isn’t very useful information. We want to understand engagement rate, so once is enough.

Comment (meltuhamy) [reply scripts/pixels.js]: Cool, I've done that. I can create a PR for the blog to fix this too.

**Comment by moollaza** [comment]
Realized we need to strip out `duckduckgo-help-pages-` from the slug as that's sometimes, but not always there. Will push a followup PR

**Pull request merged**